Examples/Python/src/Output.cpp

```diff
@@ -130,10 +131,17 @@ void addOutput(Context& ctx) {
         .def(py::init<>())
         .def(py::init<int, int, int>())
         .def(py::init<double, double, double>())
-        .def(py::init<std::string>())
+        .def(py::init<std::string_view>())
         .def_readonly("rgb", &Color::rgb);
   }
 
+  py::class_<IVisualization3D>(m, "IVisualization3D")
+      .def("write", [](const IVisualization3D& self, const py::object& arg) {
```

Comment:
This actually prevents materials from working. I need to change this back to path.